pkg/universe.dagger.io/x/ezequiel@foncubierta.com/terraform/run.cue

```diff
@@ -0,0 +1,102 @@
+package terraform
+
+import (
+  "dagger.io/dagger"
+  "dagger.io/dagger/core"
+  "universe.dagger.io/docker"
+)
+
+_#logLevel: "off" | "info" | "warn" | "error" | "debug" | "trace"
+
+// Run `terraform CMD`
+#Run: {
+  // Hashicorp Terraform container
+  containerRef: dagger.#Ref | *"hashicorp/terraform:latest"
```

Comment:
You should use a stable version instead of `latest`, 
And create a `terraform.#Image` definition would be a good idea to predefine an image or help user provide his own instead of giving a container to pull.
You can take inspiration from [`go.#Image`](https://github.com/dagger/dagger/blob/main/pkg/universe.dagger.io/go/image.cue)